MacrobenchmarkSample/app/src/main/java/com/example/macrobenchmark/target/util/Login.kt

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2022 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.macrobenchmark.target.util
+
+abstract class Login {
+    abstract suspend fun isLoggedIn(): Boolean
+    abstract suspend fun login(): Boolean
```

Comment:
should this accept username and password params and set them in the implementation?